lib/views/past_student.dart

```diff
@@ -37,6 +37,7 @@ class PastStudentState extends State<PastStudent> {
   void _sentDataToCloud() async {
     bool sendToServer = await sendDataToServer();
     if (sendToServer) {
+      await db.archiveStudentDao.deleteall();
       if (mounted) {
```

Comment:
when sendToServer is true, this means that the data is successfully uploaded to server and we are safe to delete the data from SQLite database.